packages/phrases/src/locales/en/translation/admin-console/organization-details.ts

```diff
@@ -26,19 +26,22 @@ const organization_details = {
   custom_data_tip:
     'Custom data is a JSON object that can be used to store additional data associated with the organization.',
   invalid_json_object: 'Invalid JSON object.',
-  membership_policies: 'Membership policies',
-  membership_policies_description:
-    'Define how users can join this organization and what requirements they must meet for access.',
   jit: {
-    title: 'Enable just-in-time provisioning',
+    title: 'Just-in-time provisioning',
     description:
-      'Users can automatically join the organization and receive role assignments if their verified email matches specific domains, either during sign-up or when added via the Management API.',
-    email_domains: 'JIT provisioning email domains',
-    email_domains_placeholder: 'Enter email domains for just-in-time provisioning',
+      'Users can automatically join the organization and be assigned roles upon their first sign-in through some authentication methods. You can set requirements to meet for just-in-time provisioning.',
+    email_domain: 'Email domain provisioning',
+    email_domain_description:
+      'New users signing up with their verified email addresses or through social sign-in with verified email addresses will automatically join the organization.',
+    email_domain_placeholder: 'Enter email domains for just-in-time provisioning',
     invalid_domain: 'Invalid domain',
-    sso_email_domain_description:
-      '<Icon /> means the domain is enabled for enterprise SSO. Users who signed in through the configured IdP can automatically join the organization.',
     domain_already_added: 'Domain already added',
+    sso_enabled_domain_warning:
+      'You have entered one or more email domains associated to enterprise SSO. Users with these emails will follow the standard SSO flow and won’t be provisioned to this organization unless enterprise SSO provisioning is configured.',
```

Comment:
nit
```suggestion
      'You have entered one or more email domains associated to enterprise SSO. Users with these emails will follow the standard SSO flow and won’t be provisioned to this organization unless enterprise SSO provisioning is properly configured.',
```
Btw, it seems the warning message is a little bit different from the screenshot. Seems this is a revised version, and I like this one better

Reply:
yes, didn't update the description

Reply:
added a note.